docs/configuration/configuration.md

```diff
@@ -719,6 +719,8 @@ Available meta labels:
 * `__meta_kubernetes_service_port_name`: Name of the service port for the target.
 * `__meta_kubernetes_service_port_number`: Number of the service port for the target.
 * `__meta_kubernetes_service_port_protocol`: Protocol of the service port for the target.
+* `__meta_kubernetes_service_cluster_ip`: The cluster ip of the service.
```

Comment:
These are already there

Reply:
Sorry, I just notice it. The website don't list them.